src/Core/src/Handlers/WebView/WebViewHandler.iOS.cs

```diff
@@ -17,6 +17,14 @@ public static void MapSource(WebViewHandler handler, IWebView webView)
 			handler.NativeView?.UpdateSource(webView, webViewDelegate);
 		}
 
+		public static void MapEval(WebViewHandler handler, IWebView webView, object? arg)
+		{
+			if (arg is not string script)
+				return;
+
+			handler.NativeView?.EvaluateJavaScriptAsync(script);
```

Comment:
This is scary with the swallowing of exceptions... I wonder if our mapper needs to also support async things? How are we going to get values back?

Reply:
In some cases would be necessary but no in this case: https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Platform.iOS/Renderers/WkWebViewRenderer.cs#L588